### ISSUE TITLE
APRS message in the DashBoard may exceed width of the "LAST HEARD" WebUI frame

### ISSUE THREAD

**Issue by lyusupov** [nakhonthai/ESP32APRS_T-TWR#3]
<img width="942" alt="image" src="https://github.com/nakhonthai/ESP32APRS_T-TWR/assets/5849637/e8a0e4db-808d-4287-98e6-4e418a124111">

<br>
<br>

### Expected behavior

It is expected that the frame width should stay within the style defined boundaries regardless the actual length of the APRS message

**Comment by nakhonthai** [comment]
Error in APRS packet not path or last path fail.

**Comment by lyusupov** [comment]
> Error in APRS packet not path or last path fail.

Not necessarily.

The **same source** produces the APRS packets that give NO error : 

<img width="515" alt="image" src="https://github.com/nakhonthai/ESP32APRS_T-TWR/assets/5849637/57fa556b-452c-4378-9839-a9ee460822b6">

The effect is sporadic - most packets received do not cause the ESP32APRS WebUI glitch, but some does.
I would estimate 80% PASS and 20% FAIL ratio.

Anyway, the **reason for opening this issue** is:
**regardless of the data received** - the ESP32APRS WebUI is expected to **maintain the constant HTML frames width**.

**Comment by nakhonthai** [comment]
`AC7574>OGFLR:/065718h56` <<- The package has no path.

Function handle_lastHeard() in webservice.cpp not protection code.
```
int start_info = line.indexOf(":", 0); 
int end_ssid = line.indexOf(",", 0);
int start_dst = line.indexOf(">", 2);
int start_dstssid = line.indexOf("-", start_dst);
String path = line.substring(end_ssid + 1, start_info);
```

**Comment by lyusupov** [comment]
> The package has no path.

What is wrong with an APRS message having no PATH ?
https://blog.aprs.fi/2020/02/how-aprs-paths-work.html

Moreover, the **aprs.glidernet.org** APRS-IS server that handles the **OGFLR** destination
is expecting that a receiver _(OGN project software of a generic APRS iGate)_ can process no-PATH messages:
https://github.com/glidernet/ogn-aprs-protocol/blob/master/aprsmsgs.txt

Read this excerpt for details:
https://github.com/glidernet/ogn-aprs-protocol/blob/master/aprsmsgs.txt#L118-L125

**Comment by nakhonthai** [comment]
The protocol is not wrong. But my code made a mistake in not protecting this part. Resulting in incorrect display.
I fixed it in version 0.2a
You can test update by OTA.
`SETTING->SYSTEM->ABOUT->OTA Update`

**Comment by lyusupov** [comment]
> I fixed it in version 0.2a

Thank you!

V0.2 does not seem to have the "OTA Update" button that would download & install V0.2a from your site automatically.
I can see no V0.2a .bin file in [Releases](https://github.com/nakhonthai/ESP32APRS_T-TWR/releases) area that I could download and install manually (with esptool or "Firmware Update" button).


<img width="679" alt="image" src="https://github.com/nakhonthai/ESP32APRS_T-TWR/assets/5849637/9fe6997e-5375-4828-bf00-5108fff9e244">

**Comment by lyusupov** [comment]
@nakhonthai 

Thanks' for the tip! The OTA firmware update went smooth.
 
I can confirm positive effect of this commit https://github.com/nakhonthai/ESP32APRS_T-TWR/commit/964f1e38539496c31a1846a23c3072708ce74f2f

<img width="514" alt="image" src="https://github.com/nakhonthai/ESP32APRS_T-TWR/assets/5849637/1bae629b-9d1e-4fcb-b602-241e1f023916">

I no longer see an artifact specified in the issue description.
